📚 Added a disclaimer for second time signin user object (#37)
A-Tokyo/react-apple-signin-auth: README.md

export default MyAppleSigninButton;
```

##### :warning: Disclaimer
``onSuccess`` response object will contain the user object on first time attempt. Meaning if you make another signin attempt you will not get the user object. This is true for third and N-th time attempt of the same account.

> 

Reviewer: Is the user object omitted on the second or third attempt?

Author: for any signin attempt after the first.

> ##### :warning: Disclaimer
> ``onSuccess`` response object will contain the user object on first time attempt. Meaning if you make another signin attempt you will not get the user object. This is true for third and N-th time attempt of the same account.
> 

Reviewer: ```suggestion
##### Note about the response's `user` object
`onSuccess` response object will contain the user object on the first time attempt only. Meaning if you make another signIn attempt for the same account you will not get the user object.
```
